sql/moz-fx-data-shared-prod/regrets_reporter_derived/regrets_reporter_summary_v1/query.sql

```diff
@@ -13,7 +13,7 @@ WITH base_t AS (
     `moz-fx-data-shared-prod.regrets_reporter_ucs_stable.main_events_v1`,
     UNNEST(events) e
   WHERE
-    submission_timestamp = @submission_date
+    DATE(submission_timestamp) = DATE(@submission_date)
```

Comment:
Shouldn't this be enough:
```suggestion
    DATE(submission_timestamp) = @submission_date
```

If `@submission_date` is already a date anyway.
But I guess `DATE()` on a date is essentially a no-op anyway.

Reply:
yeah, it's just to be extra safe and avoid having to open another PR. I think this should be fine.